src/instructlab/configuration.py

```diff
@@ -29,6 +29,7 @@
 DEFAULT_MODEL_OLD = "merlinite-7b-lab-Q4_K_M"
 DEFAULT_MODEL = "models/merlinite-7b-lab-Q4_K_M.gguf"
 DEFAULT_MODEL_PATH = "models/merlinite-7b-lab-Q4_K_M.gguf"
+DEFAULT_EVAL_PATH = "eval_results"
```

Comment:
results may not be the best term here.  The folder will contain a lot of the intermediate states.  The actual results should probably be returned through the command.

Reply:
Going to change to `eval_data` - @RobotSail this aligns with what's in your proposal, though the pathing will need to be changed later

Reply:
@nathan-weinberg Yes this is perfect! @danmcp We will move this into a more reasonable location later.